extraction/utils/media_extraction_utils.py

```diff
@@ -13,3 +15,12 @@ def get_media_size(node):
     if node.has_attr('width') and node.has_attr('height'):
         return int(node['width']), int(node['height'])
     return 0, 0
+
+
+def invalid_url(url):
+    """Adds necessary fix and validates image URL"""
+
+    if url.startswith('//'):
+        url = 'http:{}'.format(url)
```

Comment:
Might be a noob question, but why not "https" here?

Reply:
We are just appending protocol so that url is validated and doesn't give connection error later. And http urls are easy to read because the encryption layer won't be there.